packages/NotificationCard/src/components/Header/Header.styles.js

```diff
@@ -5,8 +5,9 @@ import tokens from "@paprika/tokens";
 
 export const NotificationHeader = styled(Heading)`
   font-weight: bold;
-  letter-spacing: 0.05em;
+  letter-spacing: 0.12em;
```

Comment:
we shouldn't use `em` those are based on the closer wrapper and this can change depending each app, if you can change it to px or lets just be aware this can create a small visual bug in certain scenarios.